README.md

```diff
@@ -4,8 +4,30 @@ Run Stable Diffusion on your machine with a nice UI without any hassle!
 
 This repository provides multiple UIs for you to play around with stable diffusion:
 
+## Quick Start
+
+- Install [Taskfile](https://taskfile.dev/installation?ref=AbdBarho-stable-diffusion-webui-docker)
```

Comment:
No problem. I operate in the DevOps/CloudEngineering space and have grown to use a pattern where even the most simple projects include a simple task file (or equivalent) because it provides a simple invocation.

Totally fine if it's not for you.

For clarity, even though I've got years of experience, I still botched up running the commands the first time because I forget that the pre-args positional sensitity for `docker compose --profile foo up` is correct, while `docker compose up --profile foo` is not. That's the main reason I wrap up these commands in this way over and over. As things get added or grow, or you jump to other repos or projects, less time is spent digging for commands and instead running `task build` or such just works even if implementation changes.

That's just an explanation so you understand where I'm coming from. Totally fine if you don't prefer that. Just thought I'd offer it! 🍻

Reply:
Skaffold is great. I've used Tilt. It's a different focus IMO, but cross over a little.
I ended up using Tilt with it just calling the same underlying tasks, so you could choose your approach.

Task runners like Make, Taskfile, Mage, etc all provide the common interface for commands that then you can leverage to simplify contribution. Then you benefit from plugging those into CI systems so the same commands you run it runs. It's a great experience overall in improving the feedback loop.